scalafix-tests/unit/src/main/resources/expect/Pretty.expect

```diff
@@ -111,7 +111,7 @@
 [116:12..116:22]: test/Test.C#ByNameType. => final object ByNameType extends AnyRef { +1 decls }
 [117:11..117:13]: test/Test.C#ByNameType.m1(). => method m1(x: => Int): Int
 [117:14..117:15]: test/Test.C#ByNameType.m1().(x) => param x: => Int
-[120:16..120:28]: test/Test.C#RepeatedType# => case class RepeatedType extends AnyRef with Product with Serializable { +11 decls }
+[120:16..120:28]: test/Test.C#RepeatedType# => case class RepeatedType extends AnyRef with Product with Serializable { +12 decls }
```

Comment:
unsurprisingly, this is a revert of https://github.com/scalacenter/scalafix/commit/7c158edb7c3e96516176b5926f7b870bf969560e#diff-829dd0f329b6e441f0af0ef30cff31ed6733345856e399618d44fc8460fe5d6eR114 which seems to be caused by https://github.com/scala/scala/pull/10155